frontend/src/libs/components/chat-sidebar/chat-sidebar.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useSearch } from '#libs/hooks/hooks.js';
+
+import { Card, Icon, Search } from '../components.js';
+import styles from './styles.module.scss';
+
+const mockedChats = [
+  { id: 1, name: 'Relationship' },
+  { id: 2, name: 'Friends' },
+  { id: 3, name: 'Family' },
+  { id: 4, name: 'Work' },
+];
+
+const mockedSelectedChat = {
+  id: 1,
+};
+
+const ChatSidebar: React.FC = () => {
+  const { filteredElements, setFilter } = useSearch(mockedChats, 'name');
+  const handleFilterChange = useCallback(
+    (search: string): void => {
+      setFilter(search);
+    },
+    [setFilter],
+  );
+
+  const handleSelectChat = useCallback((id: number) => {
+    return () => {
+      mockedSelectedChat.id = id;
+      // TODO redux logic
+    };
+  }, []);
+
+  return (
+    <div className={styles['container']}>
+      <div className={styles['header']}>
+        <div className={styles['info']}>
+          <span>Chat</span>
+          <span className={styles['chat-number']}>
+            {filteredElements.length}
+          </span>
+        </div>
+        <div className={styles['plus']}>
+          <Icon name={'plus'} color="#507CEB" />
```

Comment:
it's not a correct passing color

Reply:
@v0ldemar01 should i save colors in constant at frontend`s folder enum?

Reply:
@v0ldemar01 @Liza-Veis  said the same
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/815c0e34-b6dd-4df3-ac91-231cf1fa9e12)

Reply:
@v0ldemar01 done, but not sure

Reply:
Yep, let's proceed with color